sampler/stories/default/ToggleButton.js

```diff
@@ -0,0 +1,41 @@
+import ToggleButton, {ToggleButtonBase} from '@enact/agate/ToggleButton';
+import UiButton from '@enact/ui/Button';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {action} from '@storybook/addon-actions';
+import {withInfo} from '@storybook/addon-info';
+
+import icons from './icons';
+import {boolean, select, text} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils';
+
+ToggleButton.displayName = 'Button';
+const Config = mergeComponentMetadata('Button', UiButton, ToggleButtonBase, ToggleButton);
+
+// Set up some defaults for info and knobs
+const prop = {
+	casing: ['preserve', 'sentence', 'word', 'upper'],
```

Comment:
It doesn't look like `casing` is a supported `ToggleButton` prop (yet?). We should probably add support for it in `TogleButton` - or remove this (if support isn't need in `ToggleButton`) since it's not used here.